### PR TITLE
Update count_tokens.py

### PR DESCRIPTION
**Pull request by rachelsaunders**
## Description of the change
* integrated returns into main snippets
* updated code comments for clarity and completeness
* pulled text of prompts out of the requests to generate_content

## Motivation
Alignment and clarification of docs snippets

## Type of change
Documentation - code snippets

## Checklist

- [X] I have performed a self-review of my code.
- [X] I have added detailed comments to my code where applicable.
- [X] I have verified that my change does not break existing code.
- [X] My PR is based on the latest changes of the main branch (if unsure, please run `git pull --rebase upstream main`).
- [X] I am familiar with the [Google Style Guide](https://google.github.io/styleguide/) for the language I have coded in.
- [X] I have read through the [Contributing Guide](https://github.com/google/generative-ai-python/blob/main/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://cla.developers.google.com/about).

### CODE REVIEW

**Review by logankilpatrick — APPROVED** [review]

File: samples/count_tokens.py [hunk 1]
```diff
@@ -24,16 +24,14 @@ class UnitTests(absltest.TestCase):
     def test_tokens_context_window(self):
         # [START tokens_context_window]
         model_info = genai.get_model("models/gemini-1.0-pro-001")
-        # Returns the "context window" for the model (the combined input and output token limits)
+
+        # Returns the "context window" for the model,
+        # which is the combined input and output token limits.
         print(f"{model_info.input_token_limit=}")
```
Comment (eliben) [inline samples/count_tokens.py:30]: Here the output is printed as `key=value`, whereas in most other places it's `key: value`.
Is there some system/method behind this?

File: samples/count_tokens.py [hunk 2]
```diff
@@ -42,22 +40,18 @@ def test_tokens_text_only(self):
...
         response = model.generate_content(prompt)
 
-        # Use `usage_metadata` to get both input and output token counts
-        # (`prompt_token_count` and `candidates_token_count`, respectively).
+        # On the response for `generate_content`, use `usage_metadata`
+        # to get separate input and output token counts
+        # (`prompt_token_count` and `candidates_token_count`, respectively),
+        # as well as the combined token count (`total_token_count`).
         print(response.usage_metadata)
+        # ( prompt_token_count: 11, candidates_token_count: 73, total_token_count: 84 )
```
Comment (eliben) [inline samples/count_tokens.py:52]: Note that the token count differs between the two approaches: 10 vs 11

File: samples/count_tokens.py [hunk 3]
```diff
@@ -103,52 +93,47 @@ def test_tokens_multimodal_image_inline(self):
...
         response = model.generate_content([prompt, your_image_file])
-        # Use `usage_metadata` to get both input and output token counts
-        # (`prompt_token_count` and `candidates_token_count`, respectively).
+
+        # On the response for `generate_content`, use `usage_metadata`
+        # to get separate input and output token counts
+        # (`prompt_token_count` and `candidates_token_count`, respectively),
+        # as well as the combined token count (`total_token_count`).
         print(response.usage_metadata)
+        # ( prompt_token_count: 264, candidates_token_count: 80, total_token_count: 345 )
```
Comment (eliben) [inline samples/count_tokens.py:110]: Some off-by-1 issues here: prompt token count 263 vs 264
Also 264+80 != 345

File: samples/count_tokens.py [hunk 4]
```diff
@@ -196,71 +177,63 @@ def test_tokens_cached_content(self):
...
         )
 
         model = genai.GenerativeModel.from_cached_content(cache)
 
-        # Call `count_tokens` to get input token count of the combined text and file (`total_tokens`).
-        # A video or audio file is converted to tokens at a fixed rate of tokens per second.
-        # Optionally, you can call `count_tokens` for the prompt and file separately.
         prompt = "Please give a short summary of this file."
+
+        # Call `count_tokens` to get input token count
```
Comment (eliben) [inline samples/count_tokens.py:190]: This sample uploads a text file, not a video - should the comment be amended?

**Review by eliben — COMMENTED** [review]
Drive-by review

**Pull request merged**